bitnami/discourse/templates/configmaps.yaml

```diff
@@ -10,20 +10,24 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  {{- $port:=.Values.service.port | toString }}
-  DISCOURSE_HOSTNAME: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
+  {{- $port := .Values.service.port | toString }}
+  DISCOURSE_HOST: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
+  DISCOURSE_SKIP_INSTALL: {{ ternary "yes" "no" .Values.discourse.skipInstall | quote }}
+  DISCOURSE_SITE_NAME: {{ .Values.discourse.siteName | quote }}
   DISCOURSE_USERNAME: {{ .Values.discourse.username | quote }}
   DISCOURSE_EMAIL: {{ .Values.discourse.email | quote }}
-  DISCOURSE_SITENAME: {{ .Values.discourse.siteName | quote }}
-  REDIS_HOST: {{ template "discourse.redisHost" . }}
-  REDIS_PORT_NUMBER: {{ template "discourse.redisPort" . }}
-  POSTGRESQL_HOST: {{ template "discourse.databaseHost" . }}
-  POSTGRESQL_PORT_NUMBER: {{ template "discourse.databasePort" . }}
-  DISCOURSE_POSTGRESQL_NAME: {{ template "discourse.databaseName" . }}
-  DISCOURSE_POSTGRESQL_USERNAME: {{ template "discourse.databaseUser" . }}
-  POSTGRESQL_ROOT_USER: "postgres"
-  {{- if .Values.discourse.skipInstall }}
-  DISCOURSE_SKIP_INSTALL: "yes"
+  DISCOURSE_REDIS_HOST: {{ template "discourse.redisHost" . }}
+  DISCOURSE_REDIS_PORT_NUMBER: {{ template "discourse.redisPort" . }}
+  DISCOURSE_DATABASE_HOST: {{ template "discourse.databaseHost" . }}
+  DISCOURSE_DATABASE_PORT_NUMBER: {{ template "discourse.databasePort" . }}
+  DISCOURSE_DATABASE_NAME: {{ template "discourse.databaseName" . }}
+  DISCOURSE_DATABASE_USERNAME: {{ template "discourse.databaseUser" . }}
+  POSTGRESQL_CLIENT_DATABASE_HOST: {{ template "discourse.databaseHost" . }}
+  POSTGRESQL_CLIENT_DATABASE_PORT_NUMBER: {{ template "discourse.databasePort" . }}
```

Comment:
Now database host, port username,... are kind of duplicate between DISCOURSE_DATABASE_xxxx and POSTGRESQL_CLIENT_xxxx env vars. I am not sure the reason behind this.

Reply:
The discourse envvars are the credentials for the app to connect to the DB. The postgresql-client envvars are used to enable required extensions in the Discourse database (hstore and pg_...)